browsers/safari.json

```diff
@@ -200,7 +200,7 @@
           "engine_version": "612.4.9"
         },
         "15.4": {
-          "release_date": "2022-03-15",
+          "release_date": "2021-03-14",
```

Comment:
See https://developer.apple.com/documentation/safari-release-notes/safari-15_4-release-notes but it can't be right. @jdatapple do you want to correct the release notes?